delegator/Cargo.toml

```diff
@@ -15,25 +15,18 @@ adder = { path = "adder", default-features = false, features = ["ink-as-dependen
 subber = { path = "subber", default-features = false, features = ["ink-as-dependency"] }
 accumulator = { path = "accumulator", default-features = false, features = ["ink-as-dependency"] }
 
+
 [dev-dependencies]
 ink_e2e = { version = "4.1" }
```

Comment:
Needs to be bumped to `4.2` like in other places

Reply:
done